src/analytics/api/models/events.py

```
@@ -40,239 +40,244 @@ def __init__(self, tenant):
-                f" {devices_table}.approximate_latitude AS device_latitude , "
-                f" {devices_table}.approximate_longitude  AS device_longitude , "
-                f" FROM {data_table} "
-                f" JOIN {devices_table} ON {devices_table}.device_id = {data_table}.device_id "
-                f" WHERE {data_table}.timestamp >= '{start_date}' "
-                f" AND {data_table}.timestamp <= '{end_date}' "
-                f" AND {devices_table}.device_id IN UNNEST({devices}) "
+        for mapping in pollutant_mapping:
+            if datatype == "raw" and mapping.endswith("_raw_value"):
+                pollutant_columns.append(f"ROUND({data_table}.{mapping}, {decimal_places}) AS {mapping}")
+            elif datatype == "calibrated" and mapping.endswith("_calibrated_value"):
+                pollutant_columns.append(f"ROUND({data_table}.{mapping}, {decimal_places}) AS {mapping}")
```

Ruff

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)